coursera/api.py

```diff
@@ -543,8 +544,15 @@ def extract_links_from_lecture(self,
         if assets is None:
             assets = []
 
-        links = self._extract_videos_and_subtitles_from_lecture(
-            video_id, subtitle_language, resolution)
+        try:
+            links = self._extract_videos_and_subtitles_from_lecture(
+                video_id, subtitle_language, resolution)
+        except HTTPError as ex:
```

Comment:
I think we have a different style in other functions that do similar things.
``` python
except requests.exceptions.HTTPError as exception:
```
Please be consistent across the code.